spec/one_to_many/plural/create_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class CreateChapters < PPPT::Simple::Plural::Create(Chapter); end
+
+class CreateBooksAndChapters < PPPT::OneToMany::Plural::Create(Book)
+  create_chapters CreateChapters.new
+end
+
+describe PPPT::OneToMany::Plural::Create do
+  let(:chapter_double) { instance_double('CreateChapters', call: Success(nil)) }
```

Comment:
I think it reads easier if the name suggests it being a 'perfomer', I would call it 'create_chapters' or 'create_chapters_double'